docs/documentation/_data/sidebars/main.yml

```
@@ -896,6 +896,26 @@ entries:
                     url: /modules/015-admission-policy-engine/cr.html
               - title: FAQ
                 url: /modules/015-admission-policy-engine/faq.html
+          - title: runtime-audit-engine
+            folders:
+              - title:
```

```suggestion
          - title: runtime-audit-engine
            d8Revision: ee
            folders:
              - title:
```

It is needed to show "EE only" sticker in the sidebar